Backport patch from master branch to stop transmitting when a DFS event is detected and no available channel ist found.
 
--- /dev/null
+++ on_firmware/openwrt/package/network/services/hostapd/patches/800-dfs-enter-DFS-state-if-no-available-channel-is-found.patch
@@ -0,0 +1,13 @@
+--- a/src/ap/dfs.c
++++ b/src/ap/dfs.c
+@@ -930,7 +930,9 @@ static int hostapd_dfs_start_channel_swi
+ 			wpa_printf(MSG_INFO,
+ 				   "%s: no DFS channels left, waiting for NOP to finish",
+ 				   __func__);
+-			return err;
++			hostapd_disable_iface(iface);
++			hostapd_enable_iface(iface);
++			return 0;
+ 		}
+ 
+ 		iface->freq = channel->freq;
